package.json

```diff
@@ -40,64 +38,49 @@
     "url": "git+https://github.com/codex-team/editor.js.git"
   },
   "devDependencies": {
-    "@babel/core": "^7.9.0",
-    "@babel/plugin-transform-runtime": "^7.9.0",
-    "@babel/polyfill": "^7.8.7",
-    "@babel/preset-env": "^7.9.5",
-    "@babel/preset-typescript": "^7.13.0",
-    "@babel/register": "^7.9.0",
-    "@babel/runtime": "^7.9.2",
+    "@babel/register": "^7.21.0",
+    "@codexteam/icons": "^0.3.0",
     "@codexteam/shortcuts": "^1.1.1",
     "@cypress/code-coverage": "^3.9.2",
     "@cypress/webpack-preprocessor": "^5.6.0",
     "@editorjs/code": "^2.7.0",
     "@editorjs/delimiter": "^1.2.0",
     "@editorjs/header": "^2.7.0",
+    "@editorjs/paragraph": "^2.9.0",
     "@editorjs/simple-image": "^1.4.1",
+    "@originjs/vite-plugin-require-context": "^1.0.9",
     "@types/node": "^14.14.35",
-    "@types/webpack": "^4.41.12",
-    "@types/webpack-env": "^1.15.2",
-    "babel-loader": "^8.1.0",
-    "babel-plugin-add-module-exports": "^1.0.0",
-    "babel-plugin-class-display-name": "^2.1.0",
-    "babel-plugin-istanbul": "^6.0.0",
+    "codex-notifier": "^1.1.2",
+    "codex-tooltip": "^1.0.5",
     "core-js": "3.6.5",
     "css-loader": "^3.5.3",
     "cssnano": "^4.1.10",
     "cypress": "^6.8.0",
     "cypress-intellij-reporter": "^0.0.6",
     "eslint": "^8.28.0",
     "eslint-config-codex": "^1.7.1",
-    "eslint-loader": "^4.0.2",
     "eslint-plugin-chai-friendly": "^0.7.2",
     "eslint-plugin-cypress": "^2.12.1",
-    "extract-text-webpack-plugin": "^3.0.2",
-    "license-webpack-plugin": "^2.1.4",
+    "html-janitor": "^2.0.4",
     "mkdirp": "^1.0.4",
+    "nanoid": "^4.0.1",
     "postcss-apply": "^0.12.0",
-    "postcss-import": "^12.0.1",
     "postcss-loader": "^3.0.0",
     "postcss-nested": "^4.1.2",
-    "postcss-nested-ancestors": "^2.0.0",
-    "postcss-preset-env": "^6.6.0",
+    "postcss-preset-env": "^8.0.1",
     "rimraf": "^3.0.2",
+    "rollup-plugin-license": "^3.0.1",
     "stylelint": "^13.3.3",
-    "terser-webpack-plugin": "^2.3.6",
     "ts-loader": "^7.0.1",
     "tslint": "^6.1.1",
     "typescript": "3.8.3",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11"
+    "vite": "^4.1.4",
+    "vite-plugin-babel": "^1.1.3",
+    "vite-plugin-banner": "^0.7.0",
+    "vite-plugin-css-injected-by-js": "^3.0.1"
   },
   "collective": {
     "type": "opencollective",
     "url": "https://opencollective.com/editorjs"
-  },
-  "dependencies": {
```

Comment:
Dependencies field helps user to understand what he will get in a bundle

Reply:
i don't know about custom interpretations, but "dependencies" used first of all to install additional packages.

editor.js is used as bundle which has no dependencies. that's why you can use in on the page with a single script required.

at this time "dependencies" section provides installing a bunch of packages that will not be used:

![image](https://user-images.githubusercontent.com/15259299/227731915-a70e619b-9b52-4e44-a492-d9563b4d00a6.png)